src/components/ConfigDetailsChanges/ConfigDetailsChanges.tsx

```diff
@@ -39,85 +76,136 @@ export function ConfigDetailsChanges({
   }
 
   return (
-    <div className="overflow-hidden bg-white">
+    <div
+      className="overflow-hidden bg-white cursor-pointer"
+      onClick={(e) => {
+        setOpen(true);
+      }}
+    >
+      <Modal
+        title={
+          <>
+            <Icon
+              name={changeDetails?.change_type}
+              secondary="diff"
+              className="w-5 h-auto pr-1"
+            />
+            {changeDetails?.change_type}
+          </>
+        }
+        open={open}
+        onClose={(e) => {
+          e?.stopPropagation();
+          setOpen(false);
+        }}
+        size="large"
+        bodyClass=""
+      >
+        <div
+          className="flex flex-col h-full"
```

Comment:
Why do we need this styling? should't it be part of the modal ?

Reply:
child elements layout can column wise or row wise and it depends what kind of layout we want to show inside Modal. so we can't just move this into Modal. 

I think we can assign these classes to `bodyClass` prop of `Modal`.